sections/sections-app/src/webapp/js/sections.js

```diff
@@ -259,3 +259,26 @@ function setSectionPageFocus() {
 		}
 	}
 }
+
+function initManuallyManageDialog() {
+	$( "#dialog-confirm" ).dialog({      
+	  resizable: false,
+	  height: "auto",
+	  width: 400,
+	  modal: true,      
+	  buttons: [
+	     {
+	         text: button_ok,             
+	         click: function() {
+	        	 document.getElementById('optionsForm:confirmExternallyManagedButton').click();
```

Comment:
can you use jquery format here instead of document.getElementById for consistency?

Reply:
Updated